<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>通道A气压计</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>大气压力</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>293</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>370</y>
      <width>561</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="digitCount">
     <number>15</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_unitChange">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>单位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_manu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>740</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/Start.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serial">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>串    口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_procedure">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>程    序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remote">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>远程操作 ▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_air">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>通 大 气</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_standby">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>待    机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_measure">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>测    量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_configure">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>设    置 ▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>控  制 ▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_configureChan">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>通   道</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_configureSensor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>395</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>传 感 器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_configureControl">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>控 制 器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_about">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>700</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>关    于</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remoteMachine">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>545</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>仪  器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remoteETH">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>615</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>以 太 网</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remoteSerial">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>串  口</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_step">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>560</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_step">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>步进</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setpoint">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>620</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>设置点</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_setpoint">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>8.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_range">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>690</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>量程锁定</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_range">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>690</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_controlsetpoint">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>495</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>设 置 点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_controlstep">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>步  进</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_controlrange">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>量 程 锁 定</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_accuracy">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>580</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_step_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>101</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>精度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>721</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
